unit.js

```diff
@@ -0,0 +1,66 @@
+function Unit(name){
+  this.MAXHEALTH = 100;
+  this._name = name;
+  this._health = this.MAXHEALTH;
+}
+
+Unit.prototype._level = 1;
```

Comment:
Be carefull here, as you know, Prototype is the same object for all units, they literally will have same level field